spec/lib/tasks/opendoors.rake_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+require 'rake'
+
+describe 'Update imported urls' do
```

Comment:
As of #3473 there's an easier way to set up specs for a rake.

I think it's better if you have the rake run against an actual CSV fixture, and write the expects on the work objects (e.g. their updated times / import URLs) in addition to checking opendoors_result.txt.

Reply:
Good point. I've added a test to check that the CSV file is actually processed from the file system (and indeed it turned out it ... wasn't). Got into a bit of a tangle with having to create the CSV programmatically as it needs the work ids to be correct, but you can't use let with a before all and you can't use create outside a block and creating a Work from scratch is a pain and anyway, I ended up with instance variables and it now works, even if there's probably a better way to organise this.